definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -117,5 +117,12 @@
     "CaseEventID": "allowPetitionerToAmend",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalRejection",
```

Comment:
@Dizrupto, could you please add everything that should be feature-toggled for this "DN refusal and amend" in a separate file.
Please, have a look at what I've done in this PR and follow the same naming convention: https://github.com/hmcts/div-ccd-definitions/pull/413
This will be worth it when we come to switching this feature on.
@madjava, can you please do the same for anything you might have developed for this epic?

Reply:
yep sure